src/presentation/ui/component/mappings/mappingsInfos.templ

```diff
@@ -0,0 +1,42 @@
+package componentMappings
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+)
+
+templ MappingsInfos(
```

Comment:
What MappingsInfo means? That's a wildly generic name.

Reply:
Changed to `MappingsTargetConfigs` to refer to the target configurations created for each mapping.